incubator/kafqa/templates/deployment.yaml

```diff
@@ -0,0 +1,118 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}
+  namespace: {{ .Values.namespace }}
+  labels:
+    app.kubernetes.io/name: {{ .Release.Name }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ .Release.Name }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ .Release.Name }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+      annotations:
+        prometheus.io/scrape: "true"
+        prometheus.io/path: "/metrics"
+        prometheus.io/port: "9090"
+      namespace: {{ .Values.namespace }}
+    spec:
+      containers:
+      - name: kafqa
+        imagePullPolicy: Always
+        image: {{ .Values.image.repository }}:{{ .Values.image.tag }}
+        command: ["/bin/bash","-c","go get github.com/gojekfarm/kafqa && kafqa && ls /"]
+        env:
+          - name: KAFKA_CLUSTER
+            value: {{ .Values.kafka.cluster | quote }}
+          - name: PRODUCER_KAFKA_BROKERS
+            value: {{ .Values.producer.kafkaBrokers | quote }}
+          - name: PRODUCER_TOTAL_MESSAGES
+            value: {{ .Values.producer.totalMessages | quote }}
+          - name: PRODUCER_CONCURRENCY
```

Comment:
You can add sane default values here, so by default `yml` file doesn't need to have all variables anything other than broker ids, consumer group id etc make it default.

Reply:
I don't exactly understand this. All the fields has a default value in values.yaml. So that it's easy to overridden. 
Ex: PRODUCER_CONCURRENCY = 1 

What do you expect differently ?